src/test/scala/uk/gov/nationalarchives/aggregate/processing/modules/AssetProcessingSpec.scala

```diff
@@ -0,0 +1,48 @@
+package uk.gov.nationalarchives.aggregate.processing.modules
+
+import org.mockito.ArgumentMatchers.any
+import org.mockito.MockitoSugar.{mock, verify, when}
+import org.scalatest.matchers.must.Matchers.{noException, thrownBy}
+import org.scalatest.matchers.should.Matchers.convertToAnyShouldWrapper
+import uk.gov.nationalarchives.aggregate.processing.ExternalServiceSpec
+import uk.gov.nationalarchives.aws.utils.s3.S3Utils
+import com.typesafe.scalalogging.Logger
+import org.slf4j.{Logger => UnderlyingLogger}
+
+import java.io.ByteArrayInputStream
+
+class AssetProcessingSpec extends ExternalServiceSpec {
```

Comment:
Should cover the scenario where an exception is thrown by the S3 when fails to retrieve an object as an input stream. 

See here: https://github.com/nationalarchives/tdr-aws-utils/blob/master/s3/src/test/scala/uk/gov/nationalarchives/aws/utils/s3/S3UtilsTest.scala#L51

Would expect the exception to be handled by converting to an error, and being logged, as want the Lambda to go on processing.